editor/src/core/performance/performance-regression-tests.spec.tsx

```diff
@@ -64,7 +64,7 @@ describe('React Render Count Tests -', () => {
 
     const renderCountAfter = renderResult.getNumberOfRenders()
     // if this breaks, GREAT NEWS but update the test please :)
-    expect(renderCountAfter - renderCountBefore).toMatchInlineSnapshot(`493`)
+    expect(renderCountAfter - renderCountBefore).toMatchInlineSnapshot(`483`)
```

Comment:
Why did the render counts go down? The only new rendered elements were behind a flag that was disabled by default, so this is ... interesting

Reply:
the CanvasStrategyIndicator was not behind a flag – might be that, or maybe some other component that I removed now?

Reply:
Ah it was probably that then